### PR TITLE
Test2  Issue Template

### PR DESCRIPTION
**Pull request by aasthakasera**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Issue Template
```
Comment (DreamyPhobic) [inline .github/ISSUE_TEMPLATE.md]: remove this line...

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
```
Comment (DreamyPhobic) [inline .github/ISSUE_TEMPLATE.md]: add this as a comment.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+## Expected Behavior
+
+Please describe the behavior you are expecting
```
Comment (DreamyPhobic) [inline .github/ISSUE_TEMPLATE.md]: this one also as a comment.

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Current Behavior
+
+What is the current behavior?
```
Comment (DreamyPhobic) [inline .github/ISSUE_TEMPLATE.md]: this too.

Comment (DreamyPhobic) [reply .github/ISSUE_TEMPLATE.md]: Basically all text after title sub title should be comment.

**Review by DreamyPhobic — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (aasthakasera) [reply .github/ISSUE_TEMPLATE.md]: Kindly, give some hint.

**Comment by aasthakasera** [comment]
Changes have been made , check it.

**Pull request merged**